core/modules/date/css/date-views.css

```diff
@@ -1,3 +1,23 @@
+/* Pager plugin css */
+div.date-views-pager {
```

Comment:
Can we remove the `div` from these selectors? The more general we can be here the easier it will be for contrib to override the styles where necessary.